db/repair.cc

```diff
@@ -400,17 +400,21 @@ class Repairer {
       int64_t _current_time = 0;
       status = env_->GetCurrentTime(&_current_time);  // ignore error
       const uint64_t current_time = static_cast<uint64_t>(_current_time);
+      // Only TransactionDB make use of snapshot_checker and repair doesn't
+      // currently support TransactionDB with uncommitted prepared keys in WAL.
```

Comment:
Does Repair do any compaction? Does it need to be aware of uncommitted values? If yes, should we add a TODO to upgrade repair later?